src/scss/layout/_footer.scss

```diff
@@ -1,14 +1,15 @@
 .footer {
   color: var(--white);
   background-color: var(--navy-blue);
-  padding: 40px 15px 72px;
+  padding: 48px 0 90px;
+  
 
   &__title {
     margin-bottom: 16px;
   }
 
   &__links {
-    margin-bottom: 28px;
+    column-count: 2;
```

Comment:
this should be in a `@include breakpoint(md) { column-count: 2; }` to avoid having 2 columns on the mobile viewport.